layouts/shortcodes/accordian.html

```
@@ -0,0 +1,12 @@
+<div id="faqsBlock">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)